src/start.ts

```diff
@@ -289,12 +348,13 @@ export const handler = async (_event: any = {}): Promise<any> => {
               const percentageThreshold =
                 ethers.BigNumber.from(deviationPercentage).mul(basisPoints);
```

Comment:
There's been a lot of discussion about running deviation thresholds that are less than 1%, I'm not sure if you're familiar with that requirement? You may want to prioritise implementing that next as I they are operating under the assumption that this functionality exists.

Reply:
I told Ugur about this and they've created a task to fix it.